README.md

```diff
@@ -141,34 +144,27 @@ Wait until all tasks are running before trying to access the Kubernetes API.
 You can watch the progress what was deployed so far with:
 
 ```bash
-$ watch ./dcos kubernetes plan show deploy
+$ make watch
 ```
 
 Below is an example of how it looks like when the install ran successfully:
 
 ```
+Using Kubernetes cluster: dev/kubernetes01
 deploy (serial strategy) (COMPLETE)
    etcd (serial strategy) (COMPLETE)
       etcd-0:[peer] (COMPLETE)
-   apiserver (dependency strategy) (COMPLETE)
-      kube-apiserver-0:[instance] (COMPLETE)
+   control-plane (dependency strategy) (COMPLETE)
```

Comment:
`serial strategy` instead of `(dependency strategy)`

Reply:
@hectorj2f this is actually accurate, as reported by `dcos kubernetes cluster debug` 🤔

Reply:
okay, I know why it differs, the cluster has been updated.